modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -1006,6 +1005,18 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
             }
     }
 
+    @Test
+    @WithIroha([WithManyDomains::class])
+    @Story("Check query cursor")
+    @SdkTestId("query_cursor_test")
+    fun `cursor test`(): Unit = runBlocking {
+        val domains = QueryBuilder.findAllDomains()
```

Comment:
I'd expand this test

Reply:
with what exactly? please feel free to commit as well 😃

Reply:
agreed to comment about the internal implementation for size/cursor etc

Reply:
added comment